app/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
Please, `minifyEnabled` property should be set to true, and update if necessary the proguard files.